docker-compose.yml

```diff
@@ -202,6 +202,8 @@ services:
     depends_on:
       api:
         condition: 'service_healthy'
+    ports:
+      - 3478:3478/udp
```

Comment:
I thought we want to rely on using host networking? Does that not work with docker-compose? Allocating this port won't be enough, you also need to forward the entire other port range (or specify a fixed RNG seed for the relay and forward the first couple of "random" ones that we use for allocations).

Reply:
With host networking, we should not need any port forwarding.

Reply:
Unfortunately host networking only works on Docker Engine for Linux, not the Docker Desktop we are using for Mac/Win to build the client apps.

The workaround you suggested to fix a few ports sounds like it could work! Perhaps a rand seed env var?

Reply:
Already exists :)

https://github.com/firezone/firezone/blob/82e70411ae874126bc28ab52f6c5702201a860c5/rust/relay/src/main.rs#L61-L65

Reply:
When using `RNG_SEED = 0`, the first port in my smoke test script seems to be 65460. This should be stable across operating systems.

At the moment however, we also use the same RNG for creating nonces used for authentication so depending on how you interact with the relay, the sequence of allocation ports might not be the same all the time.

Alternatively, we could add more config options to limit the range of ports to be used to make this docker friendly. Or we use them in sequence. I'd have to check whether or not the spec says anything about not making these port predictable.

Reply:
@thomaseizinger Thanks for the note, I'll give it a shot. See https://github.com/firezone/product/issues/643